tests/manage/z_cluster/nodes/test_simultaneous_ocs_node_drains.py

```diff
@@ -0,0 +1,95 @@
+import logging
```

Comment:
We are maintaining all the drains tests in https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_nodes_maintenance.py, please move this test in that file

Reply:
Moved